crm_lead_to_task/tests/test_crm_project.py

```diff
@@ -1,24 +1,24 @@
 # Copyright 2021 Tecnativa - Pedro M. Baeza
 # Copyright 2022 Tecnativa - Víctor Martínez
+# Copyright 2024 Tecnativa - Carolina Fernandez
 # License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
 
-from odoo.tests import common
+from odoo.addons.base.tests.common import BaseCommon
 
 
-class TestCrmProject(common.TransactionCase):
+class TestCrmProject(BaseCommon):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
         cls.lead = cls.env["crm.lead"].create(
             {
                 "name": "Test lead",
                 "description": "Description",
-                "email_from": "test@example.org",
-                "partner_name": "Test partner",
+                "partner_name": "Test Partner",
```

Comment:
You are still changing this upper-casing. When I put a comment, it applies to all the occurrences of the same type. It's for not repeating the same comment everywhere.

Reply:
This is necessary because when creating task from lead, it will search for partner related to lead and search has changed from odoo 16.0 to 17.0 please check this https://github.com/odoo/odoo/blob/16.0/addons/crm/models/crm_lead.py#L1797
https://github.com/odoo/odoo/blob/17.0/addons/crm/models/crm_lead.py#L1842

partner return is with upper casing

Reply:
I still don't see the relation, as the partner doesn't exist, so it's created the same way you put on `partner_name`. Can you let it as it was to see the problem in CI?

Reply:
Please check now :)

Reply:
The problem was to use a generic partner name, which collides with another existing one. I have pushed the definitive version which doesn't fail.